### PR TITLE
Add limited delta stats after running

### PR DESCRIPTION
**Pull request by dimitraka**
Signed-off-by: Dimitra Karatza <kadimitra@ece.auth.gr>

### CODE REVIEW

File: src/deltacode/cli.py [hunk 1]
```diff
@@ -63,6 +63,16 @@ def print_version(ctx, param, value):
     click.echo('DeltaCode version ' + __version__)
     ctx.exit()
 
+def print_summary(new,old,deltas_count):
```
Comment (steven-esser) [inline src/deltacode/cli.py]: Please add a space after the comma for each of these args

File: src/deltacode/cli.py [hunk 2]
```diff
@@ -91,3 +101,7 @@ def cli(new, old, json_file, all_delta_types):
 
     # generate JSON output
     write_json(deltacode, json_file, all_delta_types)
+
+    # show limited delta status
+    deltas_count = len([d for d in deltas(deltacode, all_delta_types)])
```
Comment (steven-esser) [inline src/deltacode/cli.py:119]: I would like to see the counts of each particular Delta type, not the total number of deltas.

**Review by steven-esser — CHANGES_REQUESTED** [review]

File: src/deltacode/cli.py (hunk 2 above)
Comment (dimitraka) [reply src/deltacode/cli.py:119]: So, you would like to see the counts of each type of "factor" for each delta? That is to say factors can be for example "added", "license info added", "moved" etc. and you would like to see the counts of these such factors, lets say we have 5 "added", 3 "moved" etc.

Comment (steven-esser) [reply src/deltacode/cli.py:119]: Yes, I should have clarified that :) 

You can treat Deltas with multiple factors as their own entity for the sake of the counts

Also, I would be curious to known the performance implications of generating these counts is. We may want to add this feature as a runtime option if it takes a long time on large datasets.

**Comment by steven-esser** [comment]
Closing due to stagnation.

**Pull request closed without merging**